content/consulting/index/index.json

```diff
@@ -241,7 +241,7 @@
         {
           "title": "Power Automate Process Mining",
           "description": "Process mining uncovers processes in your company that can be effectively automated with artificial intelligence, so you save costs and increase your productivity.",
-          "logo": "/images/thumbs/thumb-power-automate-icon.jpg",
+          "logo": "",
```

Comment:
Is this logo being removed intended as part of this pr?

Reply:
Mistake thanks

Reply:
fixed in latest commit